src/services/likesServices/likesServices.js

```diff
@@ -0,0 +1,36 @@
+import models from '../../database/models';
+
+class LikeService {
+  static async findLikes(accommodation, userId) {
+    const like = await models.Likes.findOne({
+      where: { accommodationId: accommodation, user_id: userId }
```

Comment:
This condition will only allow users who have liked and accommodation to view the total number of likes in an accommodation. We should allow users to view the total number of likes/dislikes on an accommodation whether they have liked that accommodation or not